THIRD-PARTY-NOTICES

```diff
@@ -31,7 +22,7 @@ Developed by:
     http://llvm.org
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of
-this software and associated documentation files (the "Software"), to deal with
+this software and associated documentation files (the ""Software""), to deal with
```

Comment:
Craptastic! I will fix that.

I was in a rush last night. I was trying to make a self-imposed deadline for a Friday night. I have just learned a lesson on that.